dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/resource/ui/UISoftwareResource.java

```diff
@@ -75,7 +75,7 @@
   @Path("/icgc-get/versions")
   @GET
   @Produces(APPLICATION_JSON)
-  public List<ArtifactFolder> getIcgcGetArtifacts() {
+  public List<Version> getIcgcGetArtifacts() {
```

Comment:
It's storage client counterpart is named getArtifacts.  I think the point was to distinguish it from the software service function that it wraps